hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/TestSnapshotChain.java

```diff
@@ -239,8 +253,10 @@ public void testDeleteSnapshot() throws Exception {
     assertEquals(snapshotID3, chainManager.previousGlobalSnapshot(snapshotID2));
   }
 
-  @Test
-  public void testChainFromLoadFromTable() throws Exception {
+  @ParameterizedTest
+  @ValueSource(booleans = {true, false})
+  public void testChainFromLoadFromTable(boolean increasingTIme)
```

Comment:
I think it is unnecessary to make it parameterized test. Just use random number generation [here](https://github.com/apache/ozone/blob/master/hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/TestSnapshotChain.java#L75) because snapshot chain is independent of snapshot creation. It could be increasing, decreasing or  mix of both.

Reply:
There are only 2 snapshots we need to test both.

Reply:
I don't think it is needed. Because randomness will cover both scenarios over the time. Anyways it is upto you.

Reply:
You are right, but one change will run either the increasing one or the decreasing one. We should cover both the scenarios for all the changes going forward.